presto/presto.py

```diff
@@ -785,7 +786,7 @@ def construct_finetuning_model(
         return model
 
     @classmethod
-    def load_pretrained(cls):
+    def load_pretrained(cls, modelpath: Union[str, Path] = default_model_path):
```

Comment:
Could we make this `model_path` to be consistent with the naming format elsewhere in the repo?

Reply:
Done!